tests/load-tests/run-max-concurrency.sh

```diff
@@ -80,6 +80,8 @@ else
         else
             jq ".maxConcurrencyReached = $t" "$output" >"$output_dir/$$.json" && mv -f "$output_dir/$$.json" "$output"
             jq '.endTimestamp = "'$(date +%FT%T%:z)'"' "$output" >"$output_dir/$$.json" && mv -f "$output_dir/$$.json" "$output"
+            errorsTotal=$(jq '.errorsTotal' "$output_dir/load-tests.json")
+            jq ".errorsTotalLast = $errorsTotal" "$output" >"$output_dir/$$.json" && mv -f "$output_dir/$$.json" "$output"
```

Comment:
@jhutar dummy question, what does `errorsTotalLast` mean?

Reply:
Oh I get it... I thing that you can just name it `.errorsTotal`. Since all the above fields (`.maxConcurrencyReached`, `.endTimestamp`) are for the last successfull run under the threashold.